manifests/03_credentials_request_aws.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-cluster-csi-drivers
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: aws-cloud-credentials
+    namespace: openshift-cluster-csi-drivers
```

Comment:
same applies to secret's name too.

Reply:
We'll need to have a coordinated way of changing the secret name here and in the CSI driver operator repo.

If I change it here, the CI job will likely fail because the CSI driver is looking for this secret name.

Reply:
Renamed to `ebs-cloud-credentials` (similar to Manila and oVirt).

It shouldn't fail CI because for the time being the CSI driver operator is creating the secret itself.